02_bubble/sarahdavy1983/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubbleSort(s []int) []int {
+	sliceCopy := append(s[:0:0], s...)
+	n := len(s)
+	swapped := true
+	for swapped {
+		swapped = false
+		for i := 1; i < n; i++ {
+			if sliceCopy[i-1] > sliceCopy[i] {
+				sliceCopy[i], sliceCopy[i-1] = sliceCopy[i-1], sliceCopy[i]
+				swapped = true
+			}
+		}
+	}
+	return sliceCopy
+}
+func insertionSort(s []int) []int {
+	sliceCopy := append(s[:0:0], s...)
+	n := len(s)
+	for i := 1; i < n; i++ {
+		j := i
+		for j > 0 {
```

Comment:
this for loop can be consensed to declare j inline:
`for j := i; j > 0; {` 
Note that you might have tried this, but maybe just needed to add the trailing `;`